docs/build.sbt

```diff
@@ -1,6 +1,6 @@
 val ScalaVersion = "2.12.6"
 
-val AkkaVersion = "2.5.16"
+val AkkaVersion: String = sys.props.getOrElse("akka.version", "2.5.16")
```

Comment:
Also using the property for the docs.

Reply:
Hum, I wonder what would happen in the case of using two distinct versions of Akka here. Wouldn't one of them be evicted (since docs project depends on some framework subprojects)?

Reply:
🤷🏼‍♂️